packages/tfm-ror51-rubygem-arel/tfm-ror51-rubygem-arel.spec

```diff
@@ -0,0 +1,82 @@
+# Generated from arel-8.0.0.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name arel
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 8.0.0
+Release: 1%{?dist}
+Summary: Arel Really Exasperates Logicians  Arel is a SQL AST manager for Ruby
+Group:   Development/Languages
+License: MIT
+URL:     https://github.com/rails/arel
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygem(minitest) >= 5.4
+BuildRequires: %{?scl_prefix_ruby}rubygem(minitest) < 6
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%description
+Arel Really Exasperates Logicians
+Arel is a SQL AST manager for Ruby. It
+1. Simplifies the generation of complex SQL queries
+2. Adapts to various RDBMSes
+It is intended to be a framework framework; that is, you can build your own
+ORM
+with it, focusing on innovative object and collection modeling as opposed to
+database compatibility and query generation.
+
+
+%package doc
+Summary:   Documentation for %{name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
```

Comment:
The ```.``` here before ```%{gem_instdir}``` is odd.

Reply:
I think maybe that's just being explicit? Or rather, to use the relative path instead of potentially the full path in the buildroot? This comes from gem2rpm so I assume it is fairly standard.

Reply:
I later saw that in the other files too so 👍